src/main/java/com/wwi21sebgroup5/cinema/entities/Genre.java

```diff
@@ -1,4 +1,4 @@
 package com.wwi21sebgroup5.cinema.entities;
 
-public class Genre {
+public enum Genre {
```

Comment:
Ist es möglich, dass ein Film mehrere Genres hat? wenn ja -> Ist es möglich eine Art "n:m" Beziehung zu der enumeration aufzubauen?

Reply:
Hast Recht, daran hab ich nicht gedacht. Ist nicht möglich, änder ich in ne Entität